docs/api/platform/ATCmdParser.md

```diff
@@ -18,7 +18,7 @@ To use the ATCmdParser, the entity creating the ATCmdParser object passes a refe
 
 ### Example 1
 
-[![View Example](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-example-atcmdparser)](https://github.com/ARMmbed/mbed-os-example-atcmdparser/blob/master/main.cpp)
+[![View Example](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-examples-docs_only/at_cmd_example/APIs_Platform/AT_CmdParser)](https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/at_cmd_example/APIs_Platform/AT_CmdParser/main.cpp)
```

Comment:
We only have one example, not two so we should combine the two sections. 
I would describe what the code snippet does but not necessarily show the expected output as this would be difficult to maintain.

Reply:
Not this PR but the link to ESP8266 is incorrect.

Reply:
@evedon I've update the README to include a brief description of the snippet and instructions for getting started. I've also changed the incorrect link to the wi-fi drivers, although i'm not sure if it all it does is present unnecessary maintenance overhead.